types.go

```diff
@@ -97,15 +97,18 @@ type AuthorizationResponse struct {
 	// the non-raw fields are the best parsings to a single standard, with
 	// loss of granularity minimized. The latter should be preferred when
 	// treating Sleet as a black box.
-	Success              bool
-	TransactionReference string
-	AvsResult            AVSResponse
-	CvvResult            CVVResponse
-	Response             string
-	ErrorCode            string
-	AvsResultRaw         string
-	CvvResultRaw         string
-	RTAUResult           *RTAUResponse
+	Success                  bool
+	TransactionReference     string
+	AvsResult                AVSResponse
+	CvvResult                CVVResponse
+	Response                 string
+	ErrorCode                string
+	AvsResultRaw             string
+	CvvResultRaw             string
+	RTAUResult               *RTAUResponse
+	RecurringDetailReference string
```

Comment:
`RecurringDetailReference` and `Alias` are fields that are both very specific to Adyen, so I'm not sure if we should define them at the top level of `AuthorizationResponse`. Would it make more sense to return a metadata map containing these values similar to how we pass an Options map in the `AuthorizationRequest` object?

Reply:
Yes I would agree with this, then we can consume that on the hail side. Lets make sure we document it accordingly

Reply:
Created map `AdyenAdditionalData`